src/test/java/com/exasol/bucketfs/jsonrpc/CreateBucketCommandIT.java

```diff
@@ -4,26 +4,46 @@
 import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.Matchers.*;
 import static org.junit.jupiter.api.Assertions.*;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
 
+import java.security.cert.X509Certificate;
+import java.util.Optional;
+import java.util.UUID;
 import java.util.concurrent.TimeoutException;
 
+import com.exasol.bucketfs.monitor.TimestampRetriever;
+import com.exasol.bucketfs.uploadnecessity.JsonRpcReadyWaitStrategy;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Tag;
 import org.junit.jupiter.api.Test;
 
 import com.exasol.bucketfs.*;
-import com.exasol.bucketfs.testutil.BucketCreator;
 
 @Tag("slow")
 // [itest->dsn~creating-new-bucket~1]
 class CreateBucketCommandIT extends AbstractBucketIT {
+    private static X509Certificate tlsCertificate;
 
-    @Test
-    void testCreatingBucketWithCheckingCertificateThrowsException() throws BucketAccessException, TimeoutException {
-        final BucketCreator bucketCreator = bucketCreator().assumeJsonRpcAvailable();
-        final CommandFactory commandFactory = bucketCreator.createCommandFactory(true);
-        final JsonRpcException exception = assertThrows(JsonRpcException.class,
-                () -> bucketCreator.createBucket(true, commandFactory));
+    @BeforeAll
+    static void beforeAll() {
+        final Optional<X509Certificate> optionalX509Certificate = EXASOL.getTlsCertificate();
+        if (optionalX509Certificate.isPresent()) {
+            tlsCertificate = optionalX509Certificate.get();
+        } else  {
+            fail("This test requires a TLS certificate to be set up in the Exasol database.");
+        }
```

Comment:
You could do this in a one-liner using `Optional.orElseThrow(()->new AssertionError("..."))`